docs/user-guide/sdh/README.md

```diff
@@ -0,0 +1,79 @@
+| **Automated and Standardized Deployments** | Relies on manual scripts or basic automation (jobs) | Automates and standardizes deployments for consistency  |
+| **End-to-End Release Management**          | Focuses on pushing changes quickly           | Manages entire release lifecycle from planning to post-release |
+
+---
+
+## Concepts
+
+Devtron's Software Distribution Hub has 2 sections:
+
+* [Tenants](./tenants.md)
+* [Release Hub](./release-hub.md)
+
+
+Feel free to familiarize yourself with the following concepts (terms) before you proceed to Software Distribution Hub.
+
+### Tenants
+
+Tenants are organizations or clients that use your software. You can think of each tenant as a separate customer. For example, if Microsoft and Google both use your software, they are considered separate tenants. Each tenant has its own environment to ensure their data and operations are kept separate from others.
+
+### Installations
+
+One installation is equivalent to one deployment of your software for a specific [tenant](#tenants). Each installation is a different instance of the software tailored to different stages of use. For example, Microsoft might have three installations: 
+
+* **Production Installation**: Where the live environment of the software runs for everyday use.
+* **Development Installation**: Used for testing new features and changes before they go live.
+* **QA Installation**: Dedicated to quality assurance testing to ensure the software works correctly before it reaches users.
+
+### Release Tracks
+
+A Release Track in Devtron is where you organize software releases, similar to a project or application. Each release within a Release Track is a unique version of your software. For example, think of Kubernetes as a release track, with "v1.28.8" as one of its releases. This helps in managing different versions and updates of a software project in a structured manner, ensuring that all versions are tracked and organized within their respective tracks.
+
+### Requirements
+
+Requirements refer to specific steps that should be taken before deployment of applications. This includes selecting specific images for each application, specifying the [release order](#release-orderstage) of applications, adding release instructions, and locking these requirements to ensure readiness before a [rollout](#rollout). 
+
+### Release Order/Stage
+
+This is a part of [requirements](#requirements) where you decide the stages in which applications are deployed. This ensures that all dependencies are met and the software is implemented correctly. For example, you might need to deploy a database update before deploying a new application feature that relies on that update. In such a case, release order ensures that the backend applications are deployed in first stage and then the frontend applications deploy as intended.
+
+### Rollout
```

Comment:
I'm a not very sure on the description provided, rolling out a release is verb?

Reply:
Yes, 'rolling out a release' is verb
But now I rephrased the whole thing...